cluster/aws/options.md

```diff
@@ -38,6 +38,10 @@ export MASTER_SIZE=c4.large
 export MINION_SIZE=r3.large
 ```
 
+If you don't specify master and minion sizes, the scripts will attempt to guess the correct size of the master and worker nodes based on `${NUM_MINIONS}`.
+In particular for clusters less than 50 nodes it will 
```

Comment:
I wonder if we should say "Currently, for clusters" instead of "In particular for clusters".  That way we're saying the default-value is "something vaguely sensible for the version you're installing", rather than claiming that <50 always be t2.micro etc.  In particular, I'm thinking about whether if we changed to the m3 series whether that would be considered a change that we should highlight, or just part of our regular ongoing improvements 

Just a nit though; this looks great.